tests/testthat/sql/postgres-explain.sql

```diff
@@ -0,0 +1,6 @@
+<SQL>
+SELECT "x", "x" + 1.0 AS "y"
+FROM "dbplyr_001"
+
+<PLAN>
+Seq Scan on dbplyr_001  (cost=0.00..1.04 rows=3 width=36)
```

Comment:
It just occurs to me that this might cause problems if postgres changes the output format across versions. But we can cross that bridge when we come to it.